geode-core/src/main/java/org/apache/geode/management/internal/cli/shell/Gfsh.java

```diff
@@ -14,38 +14,10 @@
  */
```

Comment:
Found it.  It's an option in IDEA under 'reformat code' to 'rearrange code' and 'optimize imports'.  Those were turned off, but the upgrade to 2016.3 seemed to re-enable them.